packages/debugger/src/handlers/editor.ts

```diff
@@ -89,6 +89,13 @@ export class EditorHandler implements IDisposable {
     Signal.clearData(this);
   }
 
+  /**
+   * Refresh the breakpoints display
+   */
+  refreshBreakpoints(): void {
```

Comment:
Ideally we would avoid adding a new method to the API surface, although the editor handle is probably not a heavily used extension point anyway.

But maybe there is no other way for now to know a cell has been moved without triggering the manual refresh?

Reply:
OK let's target 3.3 then :+1: